.gitignore

```diff
@@ -98,6 +101,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+uv.lock
```

Comment:
Please read the uv docs.  This document is supposed to be committed.
```suggestion
```

Reply:
I don't want to lock my dependencies, though.